docs/stdlib/array.rst

```diff
@@ -181,8 +182,11 @@ Reference
         db> select [1, 2, 3][:-2];
         {[1]}
 
-    Referencing an array slice beyond the array boundaries will result in
-    an empty array (unlike a direct reference to a specific index):
+    If your array slice boundaries do not include any valid index from the
```

Comment:
re "your": I don't quite like this style of referring to the reader. Do we have to? IMO the previous wording is succinct and fine.